experimental/packages/otlp-transformer/src/trace/internal.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { Link } from '@opentelemetry/api';
+import { SpanKind, SpanStatusCode } from '@opentelemetry/api';
+import { hrTimeToNanoseconds } from '@opentelemetry/core';
+import type { ReadableSpan, TimedEvent } from '@opentelemetry/sdk-trace-base';
+import { toAttributes } from '../common/internal';
+import { ESpanKind, EStatusCode, IEvent, ILink, ISpan } from './types';
+
+export function toSpan(
+  span: ReadableSpan,
+): ISpan {
+  return {
+    traceId: span.spanContext().traceId,
+    spanId: span.spanContext().spanId,
```

Comment:
Suggestion: Only call span.spanContext() once.
While its only slightly better on minification it saves a deference and function call for perf.